src/calculateRentalCost.js

```diff
@@ -4,7 +4,14 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  let totalCost = days * 40;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
Avoid using literals in expressions. Define constants for the number of days threshold (`7`) and the discount amount (`50`) to prevent hardcoding and make the code more maintainable.